egs/reverb/s5/local/download_se_eval_tool.sh

```diff
@@ -18,14 +18,14 @@ unzip REVERB_scores.zip -d local/REVERB_scores_source
 rm REVERB_scores.zip
 
 pushd local/REVERB_scores_source/REVERB-SPEENHA.Release04Oct/evaltools
-sed -i 's/wavread/audioread/g' prog/score_sim.m
+perl -i -pe 's/wavread/audioread/g' prog/score_sim.m
 git clone https://github.com/MuSAELab/SRMRToolbox.git
-sed -i 's/wavread/audioread/g' SRMRToolbox/libs/preprocess.m
-sed -i 's/SRMR_main/SRMR/g' prog/score_real.m
-sed -i 's/SRMR_main/SRMR/g' prog/score_sim.m
-sed -i 's/+wb\ //g' prog/calcpesq.m
-sed -i 's/pesq_/_pesq_/g' prog/calcpesq.m
-sed -ie '30d;31d' prog/calcpesq.m
+perl -i -pe 's/wavread/audioread/g' SRMRToolbox/libs/preprocess.m
+perl -i -pe 's/SRMR_main/SRMR/g' prog/score_real.m
+perl -i -pe 's/SRMR_main/SRMR/g' prog/score_sim.m
+perl -i -pe 's/+wb\ //g' prog/calcpesq.m
```

Comment:
Ah, missed this one: the space need not be quoted with the `\`, it stands for itself. But the `+` likely needs to be quoted like `\+`, because it's a regex operator (one or more). Perl will be unhappy, even though it's the first character in the line, where it can't be an operator. sed is apparently ok with that?